views/partials/templates.hbs

```diff
@@ -0,0 +1,25 @@
+<div id="templates" class="hide">
+	<li id="alerts-template" class="list-group-item list-group-item-borderless">
```

Comment:
if you're going for full modular, why not make each a partial?

Reply:
Actually I wanted to go even further and pack those component modules like this:
```
├───assets
│  ...
├───component_modules
│   ├───alerts
│   │   ├───component.js
│   │   └───template.hbs
│  ...
│   └───invasions
│       ├───component.js
│       └───template.hbs
...
```
That would require `gulp-concat` for sure.

Reply:
i mean, i think it'd make it easier in the long run for us, go for it